### PR TITLE
Fix for update issue in application edit section

### PR DESCRIPTION
**Pull request by kayathiri4**
### Related Issues
- None

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on final implementation.
- [ ] Documentation provided. (Add links if there's any)
- [ ] Unit tests provided. (Add links if there's any)
- [ ] Integration tests provided. (Add links if there's any)

### Related PRs
- None

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

**Review by thivi — APPROVED** [review]

File: apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx [hunk 1]
```diff
@@ -366,6 +366,16 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
 
     }, [ initialValues ]);
 
+    /**
+     * Set the certificate type
+     */
+    useEffect(()=> {
+        if (certificate?.type){
```
Comment (emswbandara) [inline apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx:373]: formatting issue.

File: apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx [hunk 2]
```diff
@@ -366,6 +366,16 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
...
+            setSelectedCertType(certificate?.type);
+        }
+
+    },[certificate])
```
Comment (emswbandara) [inline apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx:377]: formatting issue.

File: apps/console/src/features/applications/components/forms/inbound-saml-form.tsx [hunk 3]
```diff
@@ -181,6 +181,16 @@ export const InboundSAMLForm: FunctionComponent<InboundSAMLFormPropsInterface> =
         return allowedOptions;
     };
 
+    /**
+     * Set the certificate type
+     */
+    useEffect(()=> {
+        if (certificate?.type){
```
Comment (emswbandara) [inline apps/console/src/features/applications/components/forms/inbound-saml-form.tsx:188]: formatting issue

File: apps/console/src/features/applications/components/forms/inbound-saml-form.tsx [hunk 4]
```diff
@@ -181,6 +181,16 @@ export const InboundSAMLForm: FunctionComponent<InboundSAMLFormPropsInterface> =
...
+            setSelectedCertType(certificate?.type);
+        }
+
+    },[certificate])
```
Comment (emswbandara) [inline apps/console/src/features/applications/components/forms/inbound-saml-form.tsx:192]: formatting issue

**Pull request merged**